src/game.cpp

```diff
@@ -1297,7 +1297,7 @@ static void updateChat(Client &client, f32 dtime, bool show_debug,
  */
 struct KeyCache {
 
-	KeyCache() { populate(); }
+	KeyCache() { handler = NULL; populate(); }
```

Comment:
if this is going to have more than one statement, it needs to be expanded to a full-bodied function